### PR TITLE
Paragraph inherits from body styles

### PR DESCRIPTION
**Pull request by ashenden**

### CODE REVIEW

File: styles/content/typography/body/styles.less [hunk 1]
```diff
@@ -2,45 +2,40 @@
...
+  body {
     .element-text-size(body, null);
     .element-text-style(body);
     display: block;
 
     & when (@body-emphasis-enabled) {
 
-      &.emphasis,
-      .emphasis {
+      span.emphasis {
```
Comment (jfurrow) [inline styles/content/typography/body/styles.less:12]: Why add the `span` selector here? I suggest we keep this class agnostic and leave it as a class. In DC/OS UI, I know for sure we use `emphasis` on `a` elements as well.

edit: I can see that you've moved the element-agnostic declaration to [line 66](https://github.com/mesosphere/cnvs/pull/97/files#diff-95cf32a51dd729266ef9f0012d6992e4R66), so I'm confused about the `span.emphasis` selector here.

Comment (ashenden) [reply styles/content/typography/body/styles.less:12]: Do you see any potential downside to the generalized class?

Comment (jfurrow) [reply styles/content/typography/body/styles.less:12]: No, I'm definitely in favor of the generalized class. But this line is adding a `span` selector (so the full selector is `span.emphasis`), so it's more specific.

Comment (ashenden) [reply styles/content/typography/body/styles.less:12]: I'm in favor of it too.  What I'm asking is, in doing so, will I potentially break anything in DC/OS today?

**Pull request merged**